contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -699,4 +787,158 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         KEYS_OP_INDEX_POSITION.setStorageUint256(keysOpIndex + 1);
         emit KeysOpIndexSet(keysOpIndex + 1);
     }
+
+    function _trimUnusedNodeOperatorKeys(uint256 _nodeOperatorId) internal returns (uint64 trimmedKeys) {
```

Comment:
Remove

Reply:
make trim for unconditional reset of staking limit (to be equal to used keys)